includes/class-cron.php

```
@@ -17,334 +17,339 @@
+		 */
+		private function mark_page_as_processed( $page_id ): void {
+			$permalink = get_permalink( $page_id );
+			if ( ! $permalink ) {
+				return;
 			}
+			$url_path   = trim( wp_parse_url( $permalink, PHP_URL_PATH ), '/' );
+			$site_url   = get_option( 'siteurl' );
+			$parsed_url = wp_parse_url( $site_url );
+			$domain     = sanitize_text_field( $parsed_url['host'] . ( $parsed_url['port'] ?? '' ) );
+
+			$cache_dir = wp_normalize_path( WP_CONTENT_DIR . "/cache/wppo/{$domain}/{$url_path}" );
```

PHPStan (2.2.5)

[warning] Constant WP_CONTENT_DIR not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)